test/preload-hover.test.js

```diff
@@ -18,13 +20,14 @@ it('should set expected link', () => {
 
   const mouseover = new MouseEvent('mouseover');
   a.dispatchEvent(mouseover);
+  jest.runTimersToTime(50);
   
   const preloadLinks = document.head.querySelectorAll('link');
   expect(preloadLinks.length).toBe(1);
   expect(preloadLinks[0].href).toBe('https://www.google.com/');
 });
 
-it('should create html link with href', () => {
+it('should create html link with href after 200 milliseconds', () => {
```

Comment:
The test name is a bit out of date with the move to 50ms. I recommend changing it to "after timeout" or similar.